### ISSUE TITLE
uss browser sometimes does not work

### ISSUE THREAD

**Issue by vvvlc** [zowe/zlux-file-explorer#7]
After first login to z Application Framework  I went to uss browser. I user ID is wrong it says `/u/izuguest` but my id is `vlcvi01` 
![Snipaste_2019-04-17_13-11-23](https://user-images.githubusercontent.com/15854496/56284675-7ebc7c00-6115-11e9-90e3-ddb9d7ab5068.png)
I tested uss api it works fine
![Snipaste_2019-04-17_13-26-25](https://user-images.githubusercontent.com/15854496/56284721-a4498580-6115-11e9-8a56-1e028dedda02.png)
I logged out of UI logged it again then I got proper screen with wrong pass `/u/vlcvi01`
![Snipaste_2019-04-17_13-30-05](https://user-images.githubusercontent.com/15854496/56284798-e4106d00-6115-11e9-9a09-62955358ca13.png)
Once I changed it to `/a/vlcvi01` then it worked.
![Snipaste_2019-04-17_13-30-57](https://user-images.githubusercontent.com/15854496/56284853-0d30fd80-6116-11e9-88c2-f6dcd558b041.png)

Bottom line is:
* wrong used id `izuguest` , why?
Vitek (Broadcom)

**Comment by DivergentEuropeans** [comment]
Hey Vitek,

Sorry, I couldn't get to this sooner. That screenshot looks like you are using the USS Explorer https://github.com/zowe/explorer-uss (not the File Explorer). That being said, I do not have too much experience with the USS Explorer and thus wouldn't be too much help though I would recommend just adding an issue there. I have no idea why they would assume a certain id upon startup. Also, it appears that is just a navigation bar (not a login of any kind) so you can just insert your own directory in USS (for ex. your user home directory is /a/vlcvi01 so ideally, USS Explorer would use your desktop credentials to issue the calls to those directories and files so you can modify them as need be). From a desktop perspective, it would probably be difficult to figure out what your home directory is given a login, so that's probably why they default to that.

That being said, I would recommend trying the **Code Editor** https://github.com/zowe/zlux-editor Just recently, we have embedded my File Explorer component into the Zowe Code Editor, and you could attempt to open your home directory. From what I understand, we are getting to a point where the Code Editor is going to be faster/more feature-rich than the MVS & USS Explorer (Code Editor combines the functionality of both USS & MVS/datasets + features like basic real-time USS fetching while you're using it). It's in an alpha-like state but every week we are adding improvements to it.

-Leanid

**Comment by DivergentEuropeans** [comment]
Hey @vvvlc Going to close this for now because of my comment from April.
